src/pages/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const Statistics = () => {
+  const [contributorDetails, setContributorsDetails] = useState(null);
+
+  useEffect(() => {
+    axios
+      .get('https://api.github.com/repos/ijemmao/igbo_api/contributors')
```

Comment:
nit: let's move this URL into the config (constants) file and then import it back into this file

Reply:
Yeah, done.

Reply:
Yeah, done.